.changeset/smart-lions-joke.md

```diff
@@ -1,7 +1,7 @@
 ---
 "@mds-core/mds-agency": minor
 "@mds-core/mds-attachment-service": minor
-"@mds-core/mds-audit": minor
+"@mds-core/mds-audit-api": minor
```

Comment:
does the version get +1.1 when it's both a minor and a patch?

Reply:
I'm pretty sure it just rolls up to a minor, so `x.y+1.z` instead of `x.y+1.z+1`